schema-extended/arxiv-extended.json

```diff
@@ -417,11 +400,26 @@
         "pattern": "^(([0-9]{4}\\.[0-9]{4,5})|([a-z\\-]+\\/[0-9]{2}[01][0-9]{4}))v[0-9]+$"
       },
       "version": {
-
         "type": "object",
         "properties": {
-          "identifier": {
-            "$ref": "#/definitions/versioned_identifier"
+
+          "identifiers": {
+            "title": "arXiv-issued identifiers for this version.",
+            "type": "object",
+            "properties": {
+                "arxiv_idv": {
+                  "title": "arXiv identifier",
+                  "description": "The arXiv identifier for this version, with a version affix.",
+                  "$ref": "#/definitions/versioned_identifier"
+                },
+                "arxiv_doi": {
+                  "title": "arXiv DOI",
+                  "description": "The arXiv-issued DOI for this version.",
+                  "$ref": "#/definitions/doi"
+                }
+            },
+            "required": ["arxiv_idv"],
+            "additionalProperties": false
           },
```

Comment:
I don't think we've firmly decided whether we'll issue DOIs for individual versions, but keeping the option open here.